netlify/sitemap-index.xml

```diff
@@ -3,4 +3,13 @@
   <sitemap><loc>https://docs.projectcalico.org/release-legacy-sitemap.xml</loc></sitemap>
   <sitemap><loc>https://docs.projectcalico.org/latest-sitemap.xml</loc></sitemap>
   <sitemap><loc>https://docs.projectcalico.org/master/sitemap.xml</loc></sitemap>
+  <sitemap><loc>https://docs.projectcalico.org/archive/v3.20/sitemap.xml</loc></sitemap>
```

Comment:
I am not sure how the archives path works, so wonder if this also needs v3.21 added?